### PR TITLE
Cleanup/normalization

### PR DESCRIPTION
**Pull request by GeBo1**
- changed `Configuration List` to `Settings List`
- standardized on `(L)`/`(R)` vs `(Left)`/`(Right)` in maker lists
- uncommented a nickname entry
- Changed `なし` back from `Nothing` to `None` since that's consistent with the other translation projects (which makes keeping things in sync a lot easier). @DarkPulse if you feel strongly it should be `Nothing` let's discuss.

### CODE REVIEW

**Review by ManlyMarco — APPROVED** [review]
None fits better imo, so I would leave that as it is now.

I'm not sure about the highres split but if it works it's probably fine, just make sure there's no regressions in one resolution vs other.

File: Translation/en/Text/Main/CharacterMaker.highres.txt [hunk 1]
```diff
@@ -0,0 +1,49 @@
+//*******************************************************************
+//*******************************************************************
+// Note: This file contains only UI elements for Chara Maker
+//       that need special handling based on resolution
+//*******************************************************************
+//*******************************************************************
+
+//Scope of the file: Only applies to the Chara Maker for Koikatsu Sunshine main game in resolutions >= 1080p
+
+#set required-resolution width > 1919
```
Comment (SpockBauru) [inline Translation/en/Text/Main/CharacterMaker.highres.txt:10]: I don't like the idea to implement this now, it's clearly a bug in XUA. I prefer to wait for a proper fix before using the new feature

**Review by SpockBauru — COMMENTED** [review]
I don't like the idea to implement the new XUA resolution feature right now, it seems buggy and may give us headaches in future.

**Comment by DarkPulse** [comment]
I figured changing it to "Nothing" makes better context in some cases.

For example, with most clothing, if they're missing, you'd say "they're wearing nothing." On the other hand, if they're missing a tattoo or don't have a mole or whatever, then "none" might be more appropriate.

Way back when we began translating KK, I went with "None" in the first place because, of course, in the early days of KK modding, there wasn't any sort of way for per-level scopes or anything like that - all strings were global, and so all strings had to be compromise. "None" was the one that would fit most generally, so I went with that. Then when per-level scopes got added by Bepis and Essu, I got to work on all the new stuff. We still didn't have per-file scope (just per-level), so "None" was stuck with (because obviously the opposite is true in other cases - "Nothing tattoos" doesn't make as much sense as "No tattoos" of course.)

It's not a dealbreaker either way (and I'd certainly say it's fine to leave the "None" icon for legibility reasons as well), and I do get the point of not wanting to break things too much since a **_LOT_** of content is shared between both games and thus both projects can help one another, but this is one of those things that's more of a polish pass thing - "None" works generally but it can be a bit awkward in some specific circumstances compared to "Nothing." Since we've got the ability to set things like this not merely per-level but on a file-by-file basis, we should take advantage of it and get the cleanest translation for it that we can, as it is not affected by having to be a global scope or even a level scope.

Of course, this is just my two cents - whatever the project goes with is what it goes with. It might well be me being too nitpicky, but my gut as an editor tells me that when it comes to clothing or items, "Nothing" would generally be more preferable to "None," as nothing implies something missing rather than simply not existing.

**Comment by GeBo1** [comment]
Ok, this should be ready now. Stuff won't work if you don't clear your cache and update your config though.

**Comment by ManlyMarco** [comment]
I think None is the way to go just to keep things consistent, both between games and within the same game. "Nothing" as an option seems weird to me, maybe because I got used to None.

**Pull request merged**